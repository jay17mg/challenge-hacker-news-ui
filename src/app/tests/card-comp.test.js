import CardComp from '../components/card-comp';
import data from './mock-data.json'
import { render } from '@testing-library/react';

describe('test card component', () =>{
    it('renders card component',() =>{
        const {container} = render(
            <>
                {/* {data.map((d,i) =>{ */}
                    <CardComp key = {data[0].id} data={data[0]} index={1} />
                {/* })} */}
            </>
        )
        // let tree = comp.toJSON()
        expect(container).toMatchSnapshot();

    }) 
})