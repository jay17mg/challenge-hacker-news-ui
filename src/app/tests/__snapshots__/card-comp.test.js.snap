// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test card component renders card component 1`] = `
<div>
  <div
    class="mb-2 smaller card"
  >
    <div
      class="card-header"
    >
      <i
        class="bi bi-caret-up-fill me-auto"
      />
      <span
        class="ms-1 badge rounded-pill bg-success"
      >
        Votes: 
        6
      </span>
      <span
        class="float-end"
      >
        <small
          class="me-1"
        >
          2 hours ago
        </small>
        <button
          class="remove-outline btn btn-outline-dark btn-sm"
          type="button"
        >
          <i
            class="text-center bi bi-x-lg"
          />
        </button>
      </span>
    </div>
    <div
      class="card-body"
    >
      <p
        class="mb-0 card-text"
      >
        2
        . 
        newest
      </p>
      <small>
        By: 
        test 1
         
      </small>
    </div>
  </div>
</div>
`;
